samples/kubernetes/end2end/mysql/mysql.yaml

```diff
@@ -18,21 +18,24 @@ spec:
         app: mysql
     spec:
       containers:
-      - image: mysql:5.6
-        name: mysql
-        env:
-        - name: MYSQL_ROOT_PASSWORD
-          value: "123456"
-        ports:
-        - containerPort: 3306
+        - image: mysql:5.6
           name: mysql
-        volumeMounts:
-        - name: mysql-persistent-storage
-          mountPath: /var/lib/mysql
+          env:
+          - name: MYSQL_ROOT_PASSWORD
```

Comment:
It looks as though you have decided to change the sublist style to use '-' indented one space from the parent, but you left this one aligned with the parent.